frame/staking/src/lib.rs

```diff
@@ -2307,19 +2314,29 @@ pub mod pallet {
 			let ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
```

Comment:
Should the origin param for `chill_other` be configurable for the pallet? (I believe @shawntabrizi mentioned this idea already somewhere) The current, permission nature of this call is useful for scenarios where we want a network to be continuously "self adjusting". But I think we can imagine scenarios where this makes more sense as a governance only call (that then can be turned into a permission-less call with a runtime upgrade).

Reply:
Tagential, is it possible to make a call origin configurable without a runtime upgrade?

Reply:
> Tagential, is it possible to make a call origin configurable without a runtime upgrade?

Nope, that's logic change and needs altering `:code`.